docs/dev/Lua API.rst

```diff
@@ -5766,82 +5766,129 @@ The names of the functions are also available as the keys of the
 building-hacks
 ==============
 
-This plugin overwrites some methods in workshop df class so that mechanical workshops are possible. Although
-plugin export a function it's recommended to use lua decorated function.
+This plugin overwrites some methods in workshop df class to change and extend workshop
+functionality. See exported function list for details.
 
 .. contents::
   :local:
 
 Functions
 ---------
 
-``registerBuilding(table)`` where table must contain name, as a workshop raw name, the rest are optional:
+* ``setOwnableBuilding(workshop_type, enable)``
 
-    :name:
-        custom workshop id e.g. ``SOAPMAKER``
+  Set workshop to be included in zones (such as bedroom or inn).
 
-        .. note:: this is the only mandatory field.
+  :workshop_type:   custom workshop string id e.g. ``SOAPMAKER`` or numeric id
```

Comment:
it might be worth mentioning, maybe in an intro to the Functions section, that numeric ids can change between worlds and a script should only pass an ID if it has looked it up for this custom workshop type in this world.